backend/de.metas.handlingunits.base/src/main/java/de/metas/handlingunits/attribute/impl/HUAttributesBL.java

```diff
@@ -86,6 +87,31 @@ public I_M_HU getM_HU(@NonNull final IAttributeSet attributeSet)
 		return hu;
 	}
 
+	@Override
+	public void updateHUAttribute(@NonNull final HuId destHUId, @NonNull final HuId sourceHUId, @NonNull final AttributeCode attributeCode)
+	{
+		final ILoggable loggable = Loggables.get();
+		final IHUStorageFactory storageFactory = handlingUnitsBL.getStorageFactory();
+		final IAttributeStorageFactory huAttributeStorageFactory = attributeStorageFactoryService.createHUAttributeStorageFactory(storageFactory);
+
+		final I_M_HU destHU = handlingUnitsDAO.getById(destHUId);
+		final I_M_HU sourceHU = handlingUnitsDAO.getById(sourceHUId);
```

Comment:
if that's the case, u could provide the I_M_HU as params

Reply:
Was hesitant to expose BL APIs using I_M_HUs instead of the more generic HuID.
But yeah, it definitely makes more sense since in the calling API I already have the I_M_HUs.